sdk/search/platform-matrix.json

```
@@ -0,0 +1,6 @@
+{
+    "matrix": {
```

This should make the test job titles a little cleaner:

```suggestion
{
    "displayNames": {
        "@{ enableCmk = $false }": "",
        "@{ enableCmk = $true }": "CMK",
    },
    "matrix": {
```